languages/yaml-language-configuration.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Not sure we need this given VSCode natively understands YAML

Reply:
It is gone